commands/sdcli_go_integration

```diff
@@ -27,7 +27,7 @@ INTTESTS=$(go test -v -tags=integration -cover -coverpkg="${PKGS}" -coverprofile
 FAIL="$(echo "${INTTESTS}" | grep 'FAIL')"
 if [[ ${FAIL} != "" ]]; then
     echo "A setup or compilation failure occurred."
-    exit 1
```

Comment:
Why should we immediately exit upon discovering a test failure? this doesn't make any sense. The exit code below https://github.com/asecurityteam/sdcli/pull/37/files#diff-a20ca3ffbc648c0766ae11176660d84910eae9fb8437ae29b2f12c2885912748L40 can handle exiting after we get our integration test failure

Reply:
This is a fine change.  You just did a better job with bash than I did!  :)